projects/gnomad/src/server/server.js

```diff
@@ -2,12 +2,21 @@ import path from 'path'
 
 import compression from 'compression'
 import express from 'express'
+import cookieParser from 'cookie-parser'
+
+const winston = require('winston');
```

Comment:
```suggestion
import winston from winston
```
Prefer import syntax